gen.py

```
@@ -33,6 +37,23 @@ def scan_images(repo_root):
     return projects
 
 
+def resolve_platform_expression(image_name, image_tags):
```

Ruff (0.15.12)

[warning] Unused function argument: `image_tags`

(ARG001)